### DISCUSSION TITLE
Something needs to be done to UTTP.

### DISCUSSION THREAD

**Discussion by Markandpreston1231** [ThioJoe/YT-Spammer-Purge#1094] [Ideas]
If you don't know, UTTP is a group that spams comments on various shorts youtubers.
I think there should be a filter which detects anything that say "MY CONTENT IS BETTER THEN (the creator's name) + BLAH BLAH BLAH + I HAVE NO LIFE")

**Comment by creeperita09** [comment]
Yeah, should also be very easy to detect and remove (just ban them if they have uttp in the username)

**Comment by virophagesp** [comment]
Text mode would make reporting them much easier but unfortunately it doesnt work if you dont own the video
I wonder why
In the meantime I am trying to make pull requests to help speed up scanning
